connectors/es/sink.py

```diff
@@ -4,19 +4,17 @@
 # you may not use this file except in compliance with the Elastic License 2.0.
 #
 """
-Implementation of BYOEI protocol (+some ids collecting)
-
 `ElasticServer` is orchestrating a sync by:
 
 - creating a queue
-- launching a `Fetcher`, a wrapper on the top of the documents' generator
-- launching a `Bulker`, a class that aggregates documents and run the bulk API
+- launching a `Extractor`, a wrapper on the top of the documents' generator
```

Comment:
From the doc string I would derive a name like `DocsGenerator`.

Reply:
"generator" is a python language term, but "DocsGenerator" would read to me as if we're fabricating the data (like Faker) as opposed to pulling the data from somewhere.